src/components/NFTPortfolio/MainView.vue

```diff
@@ -556,6 +558,12 @@ export default {
       }
       this.$emit('infinite-scroll', this.portfolioTab);
     }, 2000),
+    handleItemClick(classId) {
+      this.$emit('item-click', classId);
```

Comment:
I don't know why this emit cannot be received by the parent, still investigating...